Resources/Locale/en-US/_DV/surgery/advanced-operating-table.ftl

```diff
@@ -0,0 +1,5 @@
+advanced-operating-table-stasis-mode = It is currently [color=lightblue]slowing down decay[/color] of patients strapped to it.
+advanced-operating-table-surgery-mode = It is currently [color=lightgreen]stabilizing the patient's movement[/color].
```

Comment:
reword it, currently it implies it wouldnt make surgery faster for a corpse since it doesnt move
probably just say speeds up surgery its not a crazy secret

Reply:
Yeah, good point on both counts.